### ISSUE TITLE
Does Lime explanation support normalization form like 'NFKC' ?

### ISSUE THREAD

**Issue by rimh** [marcotcr/lime#128]
Does Lime explanation support normalization form like 'NFC' ?
I am trying to debug my text analysis with Lime. My text is written in Thai language.
As you can see according to an image below, (original text is "ไม่ รู้ สู้ หวัง")

<img width="248" alt="screen shot 2017-12-13 at 22 36 17" src="https://user-images.githubusercontent.com/19208995/33941513-47970cf4-e056-11e7-8499-8548b906bb54.png">

I believe that it is a normalization form problem but I am not sure whether display bug or scikit-learn problem. I tried to modify on scikit-learn but it did not work. When I compared with another debugging tool, eli5 with the same condition. The features list in eli5 is readable.
eli5
<img width="161" alt="screen shot 2017-12-13 at 22 36 05" src="https://user-images.githubusercontent.com/19208995/33941523-4f39dc7a-e056-11e7-9f39-c891b2a444bf.png">

Lime
<img width="884" alt="screen shot 2017-12-13 at 22 39 42" src="https://user-images.githubusercontent.com/19208995/33941608-901891b4-e056-11e7-88fa-50e2ee49ebed.png">

I like Lime very much because it is easy for the beginner like me to use. Since, eli5 does not support some modules that I want to use, I prefer to Lime for debugging. Thank you for your help in advance.

**Comment by marcotcr** [comment]
Sorry, I don't even know what NFC is.
Does the explanation look correct as a list? is this mainly a presentation problem? If so, I can take a look at it.

**Comment by rimh** [comment]
exp.as_list() and exp.show_in_notebook both are wrong.

for normalization; http://unicode.org/reports/tr15/

<img width="678" alt="screen shot 2017-12-12 at 23 44 43" src="https://user-images.githubusercontent.com/19208995/34021846-4e4bf6e8-e17f-11e7-866c-f08fc4ea5495.png">

In case the you would like to test, here is Thai words:
ไม่ รู้ ทำ หวัง น้ำ

If you could advise me for some steps to modify your codes, I will try on my own too and share the result. Thank you.

**Comment by marcotcr** [comment]
If the explanation is splitting the words incorrectly, the first thing you would need to modify the place where we split the text into tokens ([here](https://github.com/marcotcr/lime/blob/master/lime/lime_text.py#L97) and [here](https://github.com/marcotcr/lime/blob/master/lime/lime_text.py#L99)).
If that is the case, maybe you can get LIME to split it correctly without modifying the code, but using a different [split_expression parameter](https://github.com/marcotcr/lime/blob/master/lime/lime_text.py#L183).

**Comment by rimh** [comment]
Thank you for you response. I customized the split function and the words seem to be fixed but the highlights are somehow weird that make me feel worry about the analysis reliability. In my case, it is binary classification (let say for class0 and class1), 
A. without the split function:
Highlight text and feature distribution show both class0 and class1. (blue and orange highlight)
B. with the split function:
Highlight text and feature distribution show only class 0. (only orange highlight)
The rest of conditions for A and B are same.
If I understand correctly, in lime_base.py, you re-fit the model again in `explain_instance_with_data` but I do not think it will be such a critical point.

I believe that the way that I combined scikit-learn and Lime is not good enough to handle Thai language since I am new to both. Thank you again for your kindly support. If you do not mind, I would like to create PR for an example in Thai language, sometimes.
